radar-auth/build.gradle

```diff
@@ -52,42 +54,36 @@ task javadocJar(type: Jar, dependsOn: javadoc) {
     manifest.from sharedManifest
 }
 
-// add javadoc/source jar tasks as artifacts
-artifacts {
-    archives javadocJar
-    archives sourcesJar
-}
-
 publishing {
     publications {
         RadarAuthPublication(MavenPublication) {
             from components.java
             artifact sourcesJar
             artifact javadocJar
-            groupId project.group
-            artifactId project.name
-            version project.version
+
+            pom {
+                description = moduleDescription
+                name = project.name
+                url = githubUrl
+            }
+
             pom.withXml {
-                def root = asNode()
-                root.appendNode('description', description)
-                root.appendNode('name', project.name)
-                root.appendNode('url', githubUrl)
-                root.children().last() + pomConfig
+                asNode().children().last() + pomConfig
             }
         }
     }
 }
 
 bintray {
-    user = project.hasProperty('bintrayUser') ? project.property('bintrayUser') : System.getenv('BINTRAY_USER')
-    key = project.hasProperty('bintrayApiKey') ? project.property('bintrayApiKey') : System.getenv('BINTRAY_API_KEY')
-    override = false
-    publications = ['RadarAuthPublication']
+    user project.hasProperty('bintrayUser') ? project.property('bintrayUser') : System.getenv('BINTRAY_USER')
+    key project.hasProperty('bintrayApiKey') ? project.property('bintrayApiKey') : System.getenv('BINTRAY_API_KEY')
+    override false
+    publications 'RadarAuthPublication'
     pkg {
         repo = project.group
         name = project.name
         userOrg = 'radar-cns'
```

Comment:
change to `radar-base`

Reply:
That's not possible, since the bintray org is still called radar-cns.